.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Propose a new feature
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
```

Comment:
To ease navigation to users landing here, you could include a link to the discussion section in this comment

Reply:
Yes. Makes sense. I will add it as soon as it's enabled. We can hold on merging this until then.

Reply:
`https://github.com/stretchr/testify/discussions/new/choose`